migrations/0035_unitidcascade.rb

```diff
@@ -1,5 +1,6 @@
 Sequel.migration do
   up do
+    # update foreign key constraints to cascade on unit id update
```

Comment:
@bibliotechy's suggestion to add a link to the Sequel docs here is a good one, this link might be good enough?
https://github.com/jeremyevans/sequel/blob/master/doc/schema_modification.rdoc#add_foreign_key-

Reply:
agree